src/MSLGenerator.h

```diff
@@ -88,10 +88,6 @@ class MSLGenerator
     Target          m_target;
     
     bool            m_error;
-    
-    // TODO: Temporary hack to access uniforms struct members
-    HLSLBuffer* m_per_pass_buffer;
```

Comment:
Ah, I must have missed this - I thought it still used cbuffer declared in shader source, just with a hard-coded name. We use manual cbuffers on all platforms at this point (including D3D9 where they are just ignored by the compiler, and GLES 2 which is what cbuffer emulation in this commit is for). If it's hard to change the shaders to add cbuffers on certain platforms, maybe this should be an optional extension where you just specify the buffer name in some form and the translator automatically creates the necessary cbuffers?